pkgbuild/create-installer-mac.sh

```diff
@@ -23,9 +23,12 @@ fi
 set +u
 SIGN_CERT=
 SIGN_CMD=
```

Comment:
These lines should be deleted (see lines 27-28).

Reply:
good catch, updated.